### ISSUE TITLE
App doesn't seem to getting input from microphone?

### ISSUE THREAD

**Issue by FacePalmDev** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#425]
Is there a way to configure which microphone the app should be using? 
It's not picking up my default mic in a windows10 environment.

**Comment by sumeshir26** [comment]
Hi @FacePalmDev,
This feature has been released in the build(development) version and will be released as an update very soon...

**Comment by FacePalmDev** [comment]
what you mean the feature that makes it actually work?!

**Comment by sumeshir26** [comment]
@FacePalmDev Yes, but the only way to get this feature for now is to [build it yourself.](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/blob/master/CONTRIBUTING.md#how-to-build)

**Comment by FacePalmDev** [comment]
What's the point of a the project if it has no microphone though?

**Comment by FacePalmDev** [comment]
Oh I think I get it, you can still perform queries by typing right now.

**Comment by sumeshir26** [comment]
@FacePalmDev I think I misunderstood you. You can speak even in the release version by clicking this button:-
![image](https://user-images.githubusercontent.com/68823982/122111096-f0231280-cde4-11eb-9264-7fd5bacccff2.png)

**Comment by FacePalmDev** [comment]
Yes but it doesn't work. Doesn't pick up my mic

**Comment by sumeshir26** [comment]
@FacePalmDev Do you have multiple microphones attached to your computer?

**Comment by OhCrumbs** [comment]
Im having the same problem. My mic is routed through RTX Voice and it cant detect my microphone.

**Comment by Melvin-Abraham** [comment]
Possibly related to #495
